### ISSUE TITLE
custom YOLOV5 object detection model not working

### ISSUE THREAD

**Issue by maheralzoubi97** [zezo357/pytorch_lite#19]
Hello, I have installed a Yolo 5 model
The model is quite large
But it doesn't work
<img width="1085" alt="Screenshot 2023-06-02 at 2 38 03 AM" src="https://github.com/zezo357/pytorch_lite/assets/96624346/441c8ec1-0f94-4051-b78d-f97d5763db18">

**Comment by zezo357** [comment]
Did you update the labels? 


You have 80 labels? And the width and height are correct?

**Comment by maheralzoubi97** [comment]
Yes, I made sure of that
Yes, I trained the model on 1280 scale images
Do you have an idea about the problem?
@zezo357

**Comment by zezo357** [comment]
Can you send the full logs of everything,

**Comment by maheralzoubi97** [comment]
ok 
<img width="1048" alt="Screenshot 2023-06-02 at 9 36 00 AM" src="https://github.com/zezo357/pytorch_lite/assets/96624346/48a260f0-c655-41a0-b4d2-ccba84be4106">

this full logs
@zezo357

**Comment by zezo357** [comment]
> ok 
> <img width="1048" alt="Screenshot 2023-06-02 at 9 36 00 AM" src="https://github.com/zezo357/pytorch_lite/assets/96624346/48a260f0-c655-41a0-b4d2-ccba84be4106">
> 
> this full logs
> @zezo357 
> 

There is something I don't like in the first error, does the model load correctly? I want the logs for that too, it looks like the prediction can't find the model,

**Comment by maheralzoubi97** [comment]
> > ok
> > <img alt="Screenshot 2023-06-02 at 9 36 00 AM" width="1048" src="https://user-images.githubusercontent.com/96624346/242805709-48a260f0-c655-41a0-b4d2-ccba84be4106.png">
> > this full logs
> > @zezo357
> 
> There is something I don't like in the first error, does the model load correctly? I want the logs for that too, it looks like the prediction can't find the model,

<img width="777" alt="Screenshot 2023-06-02 at 9 41 17 AM" src="https://github.com/zezo357/pytorch_lite/assets/96624346/077e69e7-e391-4830-8493-871bbacbc076">

**Comment by zezo357** [comment]
Can you share all of the app logs until the error?

**Comment by maheralzoubi97** [comment]
> Can you share all of the app logs until the error?

Do you mean to send you the model and the labels?

**Comment by zezo357** [comment]
> > Can you share all of the app logs until the error?
> 
> Do you mean to send you the model and the labels?

No I mean the output of the app above the error

**Comment by maheralzoubi97** [comment]
> Can you share all of the app logs until the error?

<img width="777" alt="Screenshot 2023-06-02 at 9 41 17 AM" src="https://github.com/zezo357/pytorch_lite/assets/96624346/13468e7a-e609-4377-b608-1f4a1cae07e2">
<img width="1058" alt="Screenshot 2023-06-02 at 9 49 02 AM" src="https://github.com/zezo357/pytorch_lite/assets/96624346/d3bec0e1-402b-4e2b-8aa5-956c6ee13da8">
<img width="1054" alt="Screenshot 2023-06-02 at 9 49 19 AM" src="https://github.com/zezo357/pytorch_lite/assets/96624346/fe215025-3036-4ccb-87a7-7dc2c6ac4a11">
<img width="1055" alt="Screenshot 2023-06-02 at 9 49 34 AM" src="https://github.com/zezo357/pytorch_lite/assets/96624346/7ace54ec-17d8-4af5-bda0-bb807cbf815d">
<img width="1047" alt="Screenshot 2023-06-02 at 9 50 01 AM" src="https://github.com/zezo357/pytorch_lite/assets/96624346/2c933b3e-1d73-4944-aedd-ba18988d2d4a">

**Comment by maheralzoubi97** [comment]
@zezo357

**Comment by zezo357** [comment]
The model was not loaded correctly since it is using cuda backend did you convert it to pytorch lite?

**Comment by zezo357** [comment]
`!python export.py --weights "the weights of your model" --include torchscript --img 1280 --optimize`

**Comment by maheralzoubi97** [comment]
> The model was not loaded correctly since it is using cuda backend did you convert it to pytorch lite?

I have not personally trained the model
The model has been given to me
I tried it tflite and it works
thank you so much
I will make sure of this and tell you

@zezo357

**Comment by zezo357** [comment]
> > The model was not loaded correctly since it is using cuda backend did you convert it to pytorch lite?
> 
> I have not personally trained the model
> The model has been given to me
> I tried it tflite and it works
> thank you so much
> I will make sure of this and tell you
> 
> @zezo357 

It worked on tflite not on this package, did I get the correctly? 


You are welcome.

**Comment by maheralzoubi97** [comment]
> > > The model was not loaded correctly since it is using cuda backend did you convert it to pytorch lite?
> > 
> > 
> > I have not personally trained the model
> > The model has been given to me
> > I tried it tflite and it works
> > thank you so much
> > I will make sure of this and tell you
> > @zezo357
> 
> It worked on tflite not on this package, did I get the correctly?
> 
> You are welcome.

Yes, I tried the tflite version
Not on this package

thank you for your time
